site/.build.js

```diff
@@ -2,22 +2,36 @@ const fs = require('fs');
 const path = require('path');
 const module_path = path.join(__dirname, 'ts');
 
-function getAllFiles(dir) {
+function getTwigFiles(dir) {
```

Comment:
Maybe this function name should be more descriptive to make sure people know this is collecting the files under `ts/twig` and not under `app`